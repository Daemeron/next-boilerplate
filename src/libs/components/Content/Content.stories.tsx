import React, { Fragment } from 'react';

import { Content } from './Content';

export default {
  title: 'Components/Content',
};

export const WithText = (): JSX.Element => (
  <Fragment>
    <Content>dummy text</Content>
  </Fragment>
);
