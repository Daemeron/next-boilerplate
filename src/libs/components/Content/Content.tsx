import React, { ReactChild } from 'react';
import styled from 'styled-components';

interface ContentProps {
  children?: ReactChild | ReactChild[];
}

export const Content = (props: ContentProps): JSX.Element => <ContentBox>{props.children}</ContentBox>;

export const ContentBox = styled.div``;
