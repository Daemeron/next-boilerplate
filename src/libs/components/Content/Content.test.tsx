import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import { Content } from './Content';

describe('Content', () => {
  test('maches snapshot', () => {
    const content = render(<Content />);
    expect(content.container).toMatchSnapshot();
  });
});
