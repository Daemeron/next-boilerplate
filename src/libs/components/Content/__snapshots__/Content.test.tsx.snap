// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content maches snapshot 1`] = `
<div>
  <div
    class=""
  />
</div>
`;
